src/main/java/org/opensearch/security/dlic/rest/api/InternalUsersApiAction.java

```diff
@@ -100,6 +107,39 @@ public List<Route> routes() {
     protected Endpoint getEndpoint() {
         return Endpoint.INTERNALUSERS;
     }
+    @Override
+    protected void handleGet(final RestChannel channel, RestRequest request, Client client, final JsonNode content) throws IOException{
+        final String resourcename = request.param("name");
+
+        SecurityDynamicConfiguration<?> configuration = load(getConfigName(), true);
+        filter(configuration);
+
+        String requestDestination = request.rawPath().split("/api")[1];
```

Comment:
chnaged to split on /api instead of a spefic slash, as different requests could have different number slashes